spec/features/bag_error_notification_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+include Warden::Test::Helpers
+
+RSpec.feature 'Recieve an error notfication when creating a bag', js: true do
+  describe 'creating a notification when running a bag job' do
+    let(:work_ids) { [1, 2] }
+    let(:user) { FactoryBot.create(:admin) }
+
+    before do
+      AdminSet.find_or_create_default_admin_set_id
+      login_as user
+    end
+
+    it 'creates an error notification' do
+      BagJob.perform_now(work_ids: work_ids, user: user)
+      visit '/notifications'
+      expect(page).to have_content('error')
```

Comment:
What is it that makes the job fail?  Is it the fact that there aren't any works with ID 1 and 2 in the repo?  If that is the case, it would be nice to make the cause of the error more explicit.  Could you add something like this to the test setup?

```
# Make sure no records exist with ID 1 and 2 so that the BagJob will fail
ActiveFedora::Base.find(1).destroy(eradicate: true) if ActiveFedora::Base.exists?(1)
ActiveFedora::Base.find(2).destroy(eradicate: true) if ActiveFedora::Base.exists?(2)
```

That would also ensure that those records won't accidentally get created as part of another test.

Reply:
It's actually failing before it gets to that point with this error: `No such file or directory @ dir_s_mkdir - /home/jamie/Sync/repos/cypripedium/tmp/bags/mpls_fed_research_1543502529 Delete Notification` there's a bit of setup that's done in the other spec in the `before` and `after` blocks that I removed to get it to fail. Should I leave some comments in the spec about this?

Reply:
Yeah, maybe just write a comment about what causes it to fail - the fact that a certain directory doesn't exist - so that it's easy to tell what is meant to cause the job to fail.